src/notifications/notifications.ts

```diff
@@ -13,7 +13,7 @@ export function notifyError(title: string, details?: string) {
   emitEvent('refstudio://notifications/new', { type: 'error', title, details });
 }
 
-export function notifyErr(err: Error, title?: string, details?: string) {
+export function notifyErr(err: Error | unknown, title?: string, details?: string) {
```

Comment:
You can remove `Error`: `Error | unknown` is the same as `unknown`

Reply:
Ok. I've also added JSDoc explaining the rational